recipe/meta.yaml

```diff
@@ -52,37 +51,35 @@ requirements:
 
 outputs:
   - name: libexactreal
-{% if build_flavour != 'release' %}
-    skip: True
-{% endif %}
+    skip: True   # [build_flavour != 'release']
     script: install-libexactreal.sh
     requirements:
       build:
         # Add compiler as a build dependency again so we get a proper hash for this package.
         - {{ compiler('cxx') }}
-      run:
-        - {{ pin_compatible('e-antic') }}
+      host:
+        - {{ pin_compatible('e-antic', exact=True) }}
     run_exports:
       # We pin the version of libexactreal exactly since we do not want to
       # bother with incompatibilities in our nightly builds.
       - {{ pin_subpackage('libexactreal', exact=True) }}
+
   - name: pyexactreal
-{% if build_flavour != 'release' %}
-    skip: True
-{% endif %}
+    skip: True   # [build_flavour != 'release']
     build:
       noarch: python
     script: install-python.sh
     requirements:
       host:
-        - {{ pin_compatible('python') }}
+        - python
```

Comment:
I guess I need the same `pin_compatible` trick as with `e-antic` here. The problem is that `./configure` figures out where the `site-packages` live and `install-python.sh` calls a `setup.py` which must go to the same directory.

Reply:
I.e., the relevant `Makefile.am` says:
```
$(PYTHON) setup.py install --prefix $(DESTDIR)$(prefix) --single-version-externally-managed --record $(DESTDIR)$(pythondir)/pyexactreal/install_files.txt --verbose
```

So the `PYTHON` here must be compatible with the `pythondir` that configure found.

Reply:
Okay. IMO, pyexactreal should be a separate package in terms of autoconf even though they live in the same repo

Reply:
And, `libexactreal`'s configure can add `pyexactreal` as a subdir or not depending on a configure option like `--with-python`. In the conda recipe, you build them separately.

Reply:
I agree that they could be two packages with separate autoconf.
Doing any conditional SUBDIRS depending on configuration switches breaks `make distcheck` typically. (Whether or not that directory is part of the dist then depends on configure.)

Reply:
Does running configure again, rebuild libexactreal?

Reply:
No. That would also be an option. But then I would also make sure that all the build-time dependencies of libexactreal are still present (otherwise configure is going to complain.)

Reply:
The latter proves your point about separate configure.ac files.

Reply:
Ah, there's [https://www.gnu.org/software/automake/manual/html_node/Subpackages.html](AC_CONFIG_SUBDIRS), that's probably what I want here.

Reply:
Yes, sorry, that's what I meant. Interestingly, I learned about it in Cernay talking to James about Digraphs. :smile:

Reply:
And to fix conditional `SUBDIRS`, there's `DIST_SUBDIRS`. autoconf/automake never stops to surprise me.

Reply:
Let's see what happens if I change this to
```
exact-real/
-- libexactreal/
-- pyexactreal/
```
Feels a bit like building a monorepo by abusing old technology though.